disperser/cmd/apiserver/lib/apiserver.go

```diff
@@ -148,6 +149,13 @@ func RunDisperserServer(ctx *cli.Context) error {
 		})
 		blobStore := blobstorev2.NewBlobStore(bucketName, objectStorageClient, logger)
 
+		// Create listener for the gRPC server
+		addr := fmt.Sprintf("%s:%s", disperser.Localhost, config.ServerConfig.GrpcPort)
```

Comment:
```suggestion
		addr := fmt.Sprintf("0.0.0.0:%s", config.ServerConfig.GrpcPort)
```

Reply:
yuck can we get rid of this `Localhost = "0.0.0.0"` variable? Its not needed and a disturbingly bad name.

Reply:
https://github.com/Layr-Labs/eigenda/pull/2256/commits/1ba6ce3346a06e10eacbdbd276232c96f1c13c31